components/raftstore/src/store/fsm/apply.rs

```diff
@@ -1077,7 +1077,9 @@ where
 
             return self.process_raft_cmd(apply_ctx, index, term, cmd);
         }
-        // TOOD(cdc): should we observe empty cmd, aka leader change?
+
+        // we should observe empty cmd, aka leader change
```

Comment:
Inaccurate comment. Empty command doesn't have to be associated with leadership change.

Reply:
try to fix